exonum-java-binding/testkit/src/test/java/com/exonum/binding/testkit/SystemTimeProviderTest.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2019 The Exonum Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.exonum.binding.testkit;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.time.ZoneOffset;
+import java.time.ZonedDateTime;
+import org.junit.jupiter.api.RepeatedTest;
+
+class SystemTimeProviderTest {
+
+  @RepeatedTest(2)
```

Comment:
Why `Repeated`?

Reply:
The intention was to test that any invocation reports the up-to date time; I can clarify the intention or remove it.

Reply:
As it uses the factory method to get the provider, we don't actually check that the instance reports the up-to-date time.

I think it is not worth to test that subsequent invocations of getTime produce the current time, it must too simple to break.